src/main/java/com/egen/AppInitialize.java

```diff
@@ -20,6 +16,6 @@
 
 	@Override
 	protected String[] getServletMappings() {
-		return new  String[]{};
+		return new  String[]{"/"};
```

Comment:
it is a good practice to have /api in your context path

Reply:
Thank you for the feedback. While updating the project to spring Boot, I have added context path as /api and made some code refactoring too.